includes/admin.form.inc

```diff
@@ -59,6 +120,30 @@ function islandora_newspaper_admin_settings_form(array $form, array &$form_state
   return system_settings_form($form);
 }
 
+/**
+ * Implements hook_FORM_ID_validate().
+ */
+function islandora_newspaper_admin_settings_form_validate(array $form, array &$form_state) {
+  $error = array();
```

Comment:
Prob this spot is good for checking if solr module exist? (comment only, not requesting a change in this one)